eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -5883,6 +5883,45 @@ function (Location $location) {
         }
     }
 
+    /**
+     * Test loading content versions after removing exactly two drafts.
+     *
+     * Test for the issue EZP-30271:
+     * "Published version table disappears after bulk removing of drafts under edit"
```

Comment:
Please rather use `@see <link_to_JIRA_Issue>`

Reply:
Just wanted to keep existing convention: https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/Tests/ContentServiceTest.php#L104
Anyway, changed it as you requested.

Reply:
well... existing conventions are sometimes... IDE-unfriendly ;)